bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -105,7 +121,18 @@ if args.make_signal_kde:
     chi_eff_s  = data_signal['chi_eff'][idx]
     mchirp_s = data_signal['mchirp'][idx]
 
-    sample_s = numpy.vstack((numpy.log(mchirp_s), eta_s, chi_eff_s)).T
+    param0 = eval(args.fit_param[0][:])
+    param1 = eval(args.fit_param[1][:])
+    param2 = eval(args.fit_param[2][:])
+    if args.log_param == ['False', 'False', 'False']:
+        sample_s = numpy.vstack((param0, param1, param2)).T
+    elif args.log_param == ['True', 'False', 'False']:
+        sample_s = numpy.vstack((numpy.log(param0), param1, param2)).T
+    elif args.log_param == ['True', 'True', 'False']:
+        sample_s = numpy.vstack((numpy.log(param0), numpy.log(param1), param2)).T
+    else:
+        raise ValueError("invalid log_param argument, use 'True', or 'False'")
+
```

Comment:
When outputting values to the kde file, I would make sure the output file has at least all the parameters present in the template bank file - usually this comprises mass1, mass2, spin1, spin2z, but you can read the list of such parameters from the bank file.  Also, I would include the parameters used for the KDE as datasets in the output file.  (I would save the parameter values themselves not their logs.)
Then we need to also record whether the KDE was made over the parameter values or over their log: I think a nice way to record the parameter choice etc is to make an attr to record the fit param option, the value can be eg a string consisting of the param names separated by commas, and then another attr recording whether the log was used, the value can be a string with the True or False choices specified by the user.